pages/cart.html

```diff
@@ -0,0 +1,125 @@
+    <nav>
+      <div class="nav-route">
+        <a href="./index.html" class="nav-text">Home</a>
+        <a href="./pages/product.html" class="nav-text">Products</a>
+      </div>
+      <div class="logo-div">
+        <i class="fa-solid fa-bars"></i>
+        <p class="logo-name"><span class="logo-v">V</span>endee</p>
+        <i class="fa-solid fa-shoe-prints foot-icon"></i>
+        <i class="fa-solid fa-shoe-prints"></i>
+      </div>
+      <div class="nav-icons-div">
+        <div class="nav-icon-div">
+          <img src="../assets/nav-img/search.svg" alt="search-icon" />
+          <span>Search</span>
+        </div>
+        <a href="./pages/login.html">
+          <div class="nav-icon-div">
+            <img src="../assets/nav-img/user-plus.svg" alt="user-icon" />
+            <span>Log in</span>
+          </div>
+        </a>
+        <div class="nav-icon-div">
+          <img src="../assets/nav-img/heart.svg" alt="wishlist-icon" />
+          <span>Wishlist</span>
+        </div>
+      </div>
+    </nav>
+
+    <main class="cart">
+      <!-- <div class="container"> -->
+      <!-- <h3 class="h3 ta-center mb-6">Cart (1)</h3> -->
+
+      <div class="cart-container">
+        <div class="items">
+          <div class="cart-item">
+            <div class="cover-img">
+              <img
+                class="cart-item-img"
+                src="../assets/products-img/Nautica-Men-Tshirts-1.webp"
+                alt="first tshirt"
+              />
+            </div>
+            <div class="card-content">
+              <h4 class="card-title">Men Tshirt</h4>
+              <div class="product-price">
+                <span>Rs. 799</span>
+                <span>Rs. 999</span>
+                <span>(40% OFF)</span>
+              </div>
+              <p>
+                Quantity: <i class="px-1 fa-solid fa-circle-minus"></i>
+                1
+                <i class="fa-solid fa-circle-plus"></i>
+              </p>
+              <button class="btn">Remove from Cart</button>
+              <button class="btn">Move to wishlist</button>
+            </div>
+          </div>
+          <div class="cart-item">
+            <div class="cover-img">
+              <img
+                class="cart-item-img"
+                src="../assets/products-img/-Women-Kurta-Sets2-1.webp"
+                alt="first tshirt"
+              />
+            </div>
+            <div class="card-content">
+              <h4 class="card-title">Women Kurti</h4>
+              <div class="product-price">
+                <span>Rs. 1799</span>
+                <span>Rs. 2999</span>
+                <span>(40% OFF)</span>
+              </div>
+              <p>
+                Quantity: <i class="fa-solid fa-circle-minus"></i>
+                1
+                <i class="fa-solid fa-circle-plus"></i>
+              </p>
+              <button class="btn">Remove from Cart</button>
+              <button class="btn">Move to wishlist</button>
```

Comment:
Always have one Main CTA/ Button (Primary Button) beside an outline button as that differentiates the purpose of a button